tests/page-objects/display/languages.wdio.page.js

```diff
@@ -0,0 +1,62 @@
+const utils = require('../../utils');
+
+const submitButton = () => $('div.form-actions button.btn.btn-primary.ng-scope');
+const addLanguageButton = () => $('i.fa.fa-plus');
+const addLanguageModal = () => $('body.ng-scope.modal-open');
+const languageCodeInput = () => $('[ng-model="language.code"]');
+const languageNameInput  = () => $('[ng-model="language.name"]');
+const languageSubmitButton  = () => $('a.btn.submit.ng-scope.ng-binding.btn-primary');
+const applicationLink = () => $('i.fa.fa-fw.fa-home');
```

Comment:
As above, to have a more directed and unambiguous selector, please try finding the element by its text.